data/sql/updates/pending_db_world/rev_1676594413956062400.sql

```diff
@@ -0,0 +1,134 @@
+(249240, 13, 437.614, 2269.6, 114.766),
+(249240, 14, 438.826, 2278.85, 118.556),
+(249240, 15, 438.178, 2332.83, 119.533),
+(249240, 16, 430.33, 2389.52, 125.334),
+(249240, 17, 424.122, 2410.58, 132.903),
+(249240, 18, 409.936, 2426.29, 140.293),
+(249240, 19, 391.569, 2441.42, 147.378),
+(249240, 20, 372.028, 2450.85, 153.994),
+(249240, 21, 351.563, 2501.71, 158.671),
+(249240, 22, 358.341, 2519.12, 157.017),
+(249240, 23, 362.253, 2540.77, 155.273),
+(249240, 24, 393.193, 2549.76, 151.156),
+(249240, 25, 422.271, 2538.3, 151.843),
+(249240, 26, 443.763, 2554.36, 162.211),
+(249240, 27, 446.835, 2632.73, 179.099),
+(249240, 28, 487.397, 2745.97, 201.314),
+(249240, 29, 524.691, 2778.75, 213.732),
+(249240, 30, 529.078, 2770.99, 213.393),
+(249240, 31, 516.915, 2750.16, 211.931),
+(249240, 32, 474.551, 2663.26, 192.277),
+(249240, 33, 463.675, 2628.29, 184.522),
+(249240, 34, 463.409, 2570.53, 174.952),
+(249240, 35, 459.304, 2541.33, 164.87),
+(249240, 36, 465.254, 2485.88, 156.33),
+(249240, 37, 428.184, 2445.82, 139.387),
+(249240, 38, 428.213, 2414.12, 133.002),
+(249240, 39, 447.828, 2392.86, 133.544),
+(249240, 40, 454.476, 2300.08, 124.765),
+(249240, 41, 478.859, 2261.73, 129.556),
+(249240, 42, 490.589, 2224.36, 131.317),
+(249250, 1, 442.586, 3516.65, 60.8444),
+(249250, 2, 430.167, 3506.58, 62.984),
+(249250, 3, 380.869, 3507.06, 62.9783),
+(249250, 4, 364.828, 3502.97, 62.3136),
+(249250, 5, 359.136, 3482.24, 62.8538),
+(249250, 6, 350.932, 3469.66, 61.7651),
+(249250, 7, 333.588, 3465.64, 62.0532),
+(249250, 8, 332.589, 3436.69, 61.9835),
+(249250, 9, 344.644, 3424.99, 63.3763),
+(249250, 10, 346.874, 3386.66, 64.2423),
+(249250, 11, 341.957, 3361.83, 68.1707),
+(249250, 12, 354.937, 3358.04, 62.9527),
+(249250, 13, 373.619, 3362.38, 62.5949),
+(249250, 14, 395.768, 3352.65, 62.7729),
+(249250, 15, 417.721, 3338.98, 61.9029),
+(249250, 16, 438.161, 3340.84, 62.9257),
+(249250, 17, 463.486, 3377.86, 73.8103),
+(249250, 18, 466.918, 3398.15, 69.5736),
+(249250, 19, 505.131, 3430.84, 73.9158),
+(249250, 20, 508.783, 3437.74, 73.853),
+(249250, 21, 516.238, 3459.83, 74.7222),
+(249250, 22, 527.515, 3486.28, 77.0556),
+(249250, 23, 489.114, 3523.57, 65.8445),
+(249250, 24, 468.23, 3526.64, 63.9368);
+
+DELETE FROM `pool_template` WHERE `entry`=1085 AND `description` LIKE '%Vorakem%';
+INSERT INTO `pool_template` (`entry`, `max_limit`, `description`) VALUES (1085, 1, 'Vorakem Doomspeaker (18679)');
+
+DELETE FROM `pool_creature` WHERE `pool_entry`=1085 AND `description` LIKE '%Vorakem%';
+INSERT INTO `pool_creature` (`guid`, `pool_entry`, `chance`, `description`) VALUES
+(24922, 1085, 0, 'Vorakem Doomspeaker (18679)'),
+(24923, 1085, 0, 'Vorakem Doomspeaker (18679)'),
+(24924, 1085, 0, 'Vorakem Doomspeaker (18679)'),
+(24925, 1085, 0, 'Vorakem Doomspeaker (18679)');
+
+-- It's another rare with DISABLE_MOVE added to it
+UPDATE `creature_template` SET `unit_flags`=`unit_flags`&~0 WHERE (`entry` = 18679);
```

Comment:
this wont do anything. it's like 1 + 1 - 0

Reply:
Lmao I'm really going insane.
Sorry for these past few PRs >.<